### PR TITLE
Add baseline benchmarks

### PR DESCRIPTION
**Pull request by ggwpez**
CI does currently not work because of a problem in Substrate, but should, once fixed.

- Update the weights for pallet `Balances` to see that the `production` profile causes lower weights
- Add the `baseline` benchmarks to Polkadot and Kusama

related https://github.com/paritytech/bench-bot/pull/79 and https://github.com/paritytech/substrate/pull/10692

### CODE REVIEW

File: runtime/polkadot/src/lib.rs [hunk 1]
```diff
@@ -1708,7 +1708,7 @@ extern crate frame_benchmarking;
 #[cfg(feature = "runtime-benchmarks")]
 mod benches {
 	define_benchmarks!(
-		// Polkadot
+		// Polkadot
```
Comment (shawntabrizi) [inline runtime/polkadot/src/lib.rs:1711]:
```suggestion
		// Polkadot
```

**Review by shawntabrizi — APPROVED** [review]

File: runtime/kusama/src/weights/frame_benchmarking_baseline.rs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for `frame_benchmarking::baseline`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> frame_benchmarking::baseline::WeightInfo for WeightInfo<T> {
+	fn addition(_i: u32, ) -> Weight {
```
Comment (ggwpez) [inline runtime/kusama/src/weights/frame_benchmarking_baseline.rs:46]: @ reviewers:  
The ignored parameters here are a known issue; https://github.com/paritytech/substrate/issues/10638

**Review by gavofyork — APPROVED** [review]

**Comment by gavofyork** [comment]
CI failing.

File: runtime/kusama/src/weights/mod.rs [hunk 3]
```diff
@@ -15,6 +15,7 @@
 
 //! A list of the different weight modules for our runtime.
 
+pub mod frame_benchmarking_baseline;
```
Comment (shawntabrizi) [inline runtime/kusama/src/weights/mod.rs:18]:
```suggestion
```

File: runtime/polkadot/src/weights/mod.rs [hunk 4]
```diff
@@ -15,6 +15,7 @@
 
 //! A list of the different weight modules for our runtime.
 
+pub mod frame_benchmarking_baseline;
```
Comment (shawntabrizi) [inline runtime/polkadot/src/weights/mod.rs:18]:
```suggestion
```

**Comment by shawntabrizi** [comment]
We dont need to expose the `mod` since it is not used, and it will probably fix the compiler error

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**